src/components/register/story.js

```
@@ -1,8 +1,14 @@
 import React from 'react';
-import { storiesOf } from '@kadira/storybook';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions'; //eslint-disable-line
 import Register from './';
 
+const onChange = () => {
+  action('register on change');
+};
+
 storiesOf('Register')
   .add('User registration', () => (
-    <Register />
+    <Register handleChange={onChange} />
```

Why did this bit have to be added?

---

To get rid of the console errors when viewing the Register component and if you typed into the input boxes.

---

Ok but you need the actions addon then to show the action

---

Yeah, for now it's just going to silently eat it and not show it. I want to see if I can meaningfully log the input values with the addon. Need to look into it more.